integration_tests/mockApis/adjudications.ts

```diff
@@ -179,21 +181,31 @@ const stubGetYourReportedAdjudications = ({
   number = 0,
   size = 20,
   allContent = [],
+  filter = { status: null, toDate: moment().format('YYYY-MM-DD'), fromDate: moment().format('YYYY-MM-DD') },
```

Comment:
Using the current date is a bit risky for tests - can they be set to null or in fact any old value?

Actually, maybe just don't have a default at all?!

Reply:
So... the reason for this is that today is the default - so when we call the the api we would expect to get these values requested.